server/routes/allocate-to-activity/handlers/activities.ts

```diff
@@ -11,20 +10,30 @@ export default class ActivitiesRoutes {
 
   GET = async (req: Request, res: Response): Promise<void> => {
     const { user } = res.locals
-    const { categoryId } = req.params
+    const categoryId = +req.params.categoryId
 
-    const addAllocationSummary = async (a: ActivityLite) => ({
-      ...a,
-      ...(await this.capacitiesService.getActivityAllocationsSummary(a.id, user)),
-    })
+    const today = new Date()
+    today.setHours(0, 0, 0, 0)
 
-    const activities = await this.activitiesService
-      .getActivitiesInCategory(categoryId as unknown as number, user)
-      .then(c => Promise.all(c.map(addAllocationSummary)))
+    const activitiesWithAllocations = await this.activitiesService
+      .getActivitiesInCategory(categoryId, user)
```

Comment:
Just a thought but I wonder if it would be worth extending the API endpoint to include a filter/query param?  i.e. only  active activities?